model_compression_toolkit/qat/keras/quantization_facade.py

```diff
@@ -256,8 +256,16 @@ def keras_quantization_aware_training_finalize(in_model: Model) -> Model:
 
          """
         def _export(layer):
-            if isinstance(layer, (KerasQuantizationWrapper, KerasActivationQuantizationHolder)):
+            if isinstance(layer, KerasQuantizationWrapper):
                 layer.convert_to_inferable_quantizers()
+            # In the KerasActivationQuantizationHolder case - converting the quantizers only
+            # is not enough. We need to create a new layer with inferable quantizers. The reason for that
+            # is that if we only convert the quantizers, the layer will have some weights that do not
+            # match the configuration, thus loading such a model will fail. To overcome this, the
```

Comment:
Specify what are these "some weights" that the layer would have (that they are for training only)